dist/categories/create/index.html

```diff
@@ -17,16 +17,41 @@ <h1>Create categories</h1>
       <form autocomplete="off" name="create-user-form" novalidate>
         <div class="field-container">
           <label for="category-name" class="form-label">Category name</label>
-          <input class="form-control"type="text" name="category-name" />
+          <input class="form-control" type="text" name="category-name" />
           <div class="invalid-feedback hidden" name="category-name-required">
             * Campo requerido
           </div>
         </div>
-        
 
-        <button id="create-category-button" type="submit" class="btn btn-primary">Create</button>
+        <button
+          id="create-category-button"
+          type="submit"
+          class="btn btn-primary"
+        >
+          Create
+        </button>
       </form>
-    </main>        
+    </main>
+    <div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+      <div
+        id="delete-toast"
```

Comment:
Change all the ID `id="delete-toast"`. in this case to id="create-toast"
You will. have to change also the ID on the TS. files.

Reply:
Work Done